tests/test_fft_inverse_blank_forward/Makefile

```diff
@@ -1,22 +1,32 @@
+BUILD_FLAGS = -O2 -g -report -DDEBUG_PRINT_ENABLE=1
+
+XCOREAI ?= 0
+ifeq ($(XCOREAI),0)
```

Comment:
Same comment as app_adaptive Makefile.

Reply:
This has been fixed by https://github.com/xmos/lib_dsp/pull/193